bundler/spec/dependabot/bundler/file_updater/gemfile_updater_spec.rb

```diff
@@ -282,9 +292,9 @@
       context "that should be removed" do
         let(:dependency) do
           Dependabot::Dependency.new(
-            name: "business",
-            version: "1.8.0",
-            previous_version: "c5bf1bd47935504072ac0eba1006cf4d67af6a7a",
+            name: "dependabot-test-ruby-package",
+            version: "1.0.0",
```

Comment:
Nit, should this be `1.1.0` to match the test expectation? Not sure if it's actually used but more for consistency, think `requirements` is the one used to update to the new version